pages/common/dolt-init.md

```diff
@@ -0,0 +1,12 @@
+# dolt init
+
+> Create an empty Dolt data repository.
+> More information: <https://docs.dolthub.com/interfaces/cli#dolt-config>.
+
+- Create an empty Dolt data repository in the current directory:
+
+`dolt init`
+
+- Specify details name on the initial commit:
```

Comment:
```suggestion
- Specify details on the initial commit:
```

Reply:
little slip on last update, fixed in de9e53eaef9d7566340cc11ea34aeface4764d09